### PR TITLE
Disabled DeleteFromVMDB in 2 places.

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
https://bugzilla.redhat.com/show_bug.cgi?id=1425206

Modified StackRetirement Schema and Infrastructure/VM/Retirement unregister instance.

DeleteFromVMDB is disabled everywhere else.

### CODE REVIEW

**Comment by billfitzgerald0120** [comment]
@tinaafitz 
Please Review

**Comment by gmcculloug** [comment]
@billfitzgerald0120 @tinaafitz Is there a BZ you can associate to this?

**Comment by billfitzgerald0120** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1425206

**Comment by billfitzgerald0120** [comment]
@tinaafitz 
Please Review

File: content/automate/ManageIQ/Infrastructure/VM/Retirement/StateMachines/VMRetirement.class/unregister.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ object_type: instance
 version: 1.0
 object:
   attributes:
-    display_name: Unregister
+    display_name: 
     name: Unregister
     inherits: 
     description:
```
Comment (tinaafitz) [inline content/automate/ManageIQ/Infrastructure/VM/Retirement/StateMachines/VMRetirement.class/unregister.yaml]: @billfitzgerald0120 Can you comment out the DeleteFromVMDB call here?

Comment (billfitzgerald0120) [inline content/automate/ManageIQ/Infrastructure/VM/Retirement/StateMachines/VMRetirement.class/unregister.yaml]: I commented it out ...

**Comment by billfitzgerald0120** [comment]
@tinaafitz 
Please Review

**Comment by tinaafitz** [comment]
@billfitzgerald0120 Looks good.
@mkanoor Please review.

**Pull request merged**